app/src/main/java/com/example/parkingsystem/MainActivity.kt

```diff
@@ -2,10 +2,13 @@ package com.example.parkingsystem
 
 import androidx.appcompat.app.AppCompatActivity
 import android.os.Bundle
+import com.example.parkingsystem.databinding.ActivityMainBinding
 
 class MainActivity : AppCompatActivity() {
+    private lateinit var binding: ActivityMainBinding
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
-        setContentView(R.layout.activity_main)
+        binding = ActivityMainBinding.inflate(layoutInflater)
+        setContentView(binding.root)
     }
 }
```

Comment:
add an empty line at the end of the file to avoid this error pls

Reply:
to avoid this
![image](https://user-images.githubusercontent.com/56684696/128245642-20e38ec3-e6d1-44e6-aa2f-30ea8119f888.png)

Reply:
Done